sklearn/externals/_joblib.py

```diff
@@ -5,11 +5,12 @@
 
 # An environment variable to use the site joblib
```

Comment:
Maybe `sklearn/externals/_joblib.py` can be moved to `sklearn/utils/_joblib.py`.

My reasoning is that `_joblib.py` is not really external but our own code to do the switching between `sklearn.external.joblib` and `joblib`.